integration/options.go

```diff
@@ -163,6 +182,42 @@ type testOptions interface {
 
 	// VerifySeriesDebugFilePathPrefix returns the file path prefix for writing a debug file of series comparisons.
 	VerifySeriesDebugFilePathPrefix() string
+
+	// SetRepairThrottle sets minimum duration of time a namespace repair is throttled to.
+	SetRepairThrottle(throttle time.Duration) testOptions
+
+	// RepairThrottle returns the repair throttle duration.
+	RepairThrottle() time.Duration
+
+	// SetRepairInterval sets interval between repairs.
+	SetRepairInterval(interval time.Duration) testOptions
+
+	// RepairInterval returns the repair interval duration.
+	RepairInterval() time.Duration
+
+	// SetRepairTimeJitter sets the repair time jitter.
+	SetRepairTimeJitter(t time.Duration) testOptions
+
+	// RepairTimeJitter returns the repair time jitter.
+	RepairTimeJitter() time.Duration
+
+	// SetNumShards sets the number of shards.
+	SetNumShards(numShards uint32) testOptions
+
+	// NumShards returns the number of shards.
+	NumShards() uint32
+
+	// SetFetchSeriesBlocksBatchSize sets the number of Series Blocks to fetch in batch.
+	SetFetchSeriesBlocksBatchSize(n int) testOptions
+
+	// FetchSeriesBlocksBatchSize returns the number of Series Blocks to fetch in batch.
+	FetchSeriesBlocksBatchSize() int
+
+	// SetFetchSeriesBlocksBatchConcurrency sets the number of series blocks to fetch in batch concurrently.
+	SetFetchSeriesBlocksBatchConcurrency(n int) testOptions
+
+	// FetchSeriesBlocksBatchConcurrency returns the number of series blocks to fetch in batch concurrently.
+	FetchSeriesBlocksBatchConcurrency() int
```

Comment:
That makes sense, perhaps could we make the default integration test namespaces disable repairs by default?

Just so there's less overhead when we run other integration tests and we only run repairs when we are specifically testing the repair feature?